tests/client/amo/containers/TestApp.js

```diff
@@ -95,4 +95,31 @@ describe('App', () => {
     assert.equal(_window.location, 'https://a.m.org/login');
     assert.ok(startLoginUrlStub.calledWith({ location }));
   });
+
+  it('removes the mamo cookie', () => {
+    const fakeEvent = {
+      preventDefault: sinon.stub(),
+    };
+    const fakeWindow = {
+      location: {
+        reload: sinon.stub(),
+      },
+    };
+    const fakeCookieLib = {
+      remove: sinon.stub(),
+    };
+    const i18n = getFakeI18nInst();
+    const location = sinon.stub();
+    const root = renderIntoDocument(
+      <AppBase i18n={i18n}
```

Comment:
Again, just ordering stuff but we have been putting `UpperCaseComponents` before `lowerCaseProps`.